ood-portal-generator/lib/ood_portal_generator/dex.rb

```diff
@@ -34,14 +35,12 @@ def initialize(opts = {}, view)
       @dex_config[:staticClients] = static_clients
       @dex_config[:connectors] = connectors unless connectors.nil?
       @dex_config[:oauth2] = { skipApprovalScreen: true }
-      @dex_config[:enablePasswordDB] = connectors.nil?
-      if connectors.nil?
-        @dex_config[:staticPasswords] = [{
-          email: 'ood@localhost',
-          hash: '$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W',
-          username: 'ood',
-          userID: '08a8684b-db88-4b73-90a9-3cd1661f5466',
-        }]
+      configured_static_passwords = @config.fetch(:static_passwords, nil)
+      if (connectors.nil? || !configured_static_passwords.nil?)
```

Comment:
I think changing this to just `if connectors.nil?` so that once you configure LDAP, you can't use local accounts. That's how things worked before but I added one extra condition to support whatever that MockConnector is for dev container.

Reply:
Yea the MockConnector we need to get rid of in the dev container too, I think I had to define _something_ to get this library to not create the ood local user. So that's something we'd need to fix either way.